Models/articles.model.js

```diff
@@ -46,3 +46,21 @@ exports.selectArticleById = (id) => {
             return Promise.reject(error);
         });
 };
+
+exports.selectCommentsByArticle = (id) => {
+    return db
+        .query(
+            `
+            SELECT comment_id, votes, created_at, author, body, article_id
+            FROM comments
+            Where article_id = $1
+            ORDER BY created_at DESC`,
+            [id]
+        )
+        .then(({ rows }) => {
+            return rows;
+        })
+        .catch((error) => {
```

Comment:
we dont want a .catch in the model :) we only need a catch at the end of the promise chain in the controller!